tests/e2e/util/certs/certs.go

```diff
@@ -227,8 +227,8 @@ func PushIntermediateCA(ns, kubeconfig, zone, network, basePath string, cl clien
 	_, err := common.GetObject(context.Background(), cl, kube.Key("cacerts", ns), &corev1.Secret{})
 	if err != nil {
 		// Label the namespace with the network
-		k := kubectl.NewKubectlBuilder()
-		k.SetKubeconfig(kubeconfig)
+		k := kubectl.New()
+		k.WithKubeconfig(kubeconfig)
```

Comment:
I think we have an issue here. This should be
```
k = k.WithKubeconfig(kubeconfig)
```
or even better, we can remove that statement itself and modify the next line to
```
err = k.WithKubeconfig(kubeconfig).Patch(....)
```

Reply:
AAAaaaa, you're right. This is what's causing the test to fail. I've been trying to figure this out for the past two hours. 

Remind me to buy you a drink at the next F2F :)

Reply:
`WithKubeconfig()` is not only returning `k` but also updating the `k.kubeconfig` which is finally used in the actual commands, so we are good. Sure, you can update it in a different PR.